test/extensions/transport_sockets/internal_upstream/BUILD

```diff
@@ -43,6 +43,9 @@ envoy_extension_cc_test(
         "//source/extensions/access_loggers/file:config",
         "//source/extensions/bootstrap/internal_listener:config",
         "//source/extensions/filters/network/tcp_proxy:config",
+        "//source/extensions/clusters/dynamic_forward_proxy:cluster",
+        "//source/extensions/filters/http/dynamic_forward_proxy:config",
```

Comment:
General comment: we're generally trying to limit dependencies from the integration test on real filters. Maybe create another test file to keep it separate (and I think this could live in internal_listener test suite).

Reply:
+1 integration tests should only rely on "core" extensions. For these tests DFP does not really make any difference. Please change the test to not use DFP.